Week1/homework/src/results.js

```diff
@@ -0,0 +1,181 @@
+'use-strict';
+
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world',
+});
+
+function one() {
+  connection.connect();
+  connection.query(
+    'SELECT name, population FROM countries WHERE population > 8000000 LIMIT 20',
+    function(error, results) {
+      if (error) throw error.message;
+      console.log(`
+        1- Countries with population greater than 8 million:
+        `);
+      results.forEach(country => console.log(JSON.parse(JSON.stringify(country))));
+      console.log(`
+        `);
+    }
+  );
+  connection.end();
+}
+
+function two() {
+  connection.connect();
+  connection.query('SELECT name FROM countries WHERE name LIKE "%land%" LIMIT 20', function(
```

Comment:
You don't have to limit here. A small thing: could there be countries with the same name? How do you deal with that?

Reply:
Dear @zhaobn ,
I used limit because I am fetching country data from an API. And if I don't limit, it will be many rows and will be hard to see all of the data.